site/cypress/e2e/pdfs.spec.js

```diff
@@ -0,0 +1,102 @@
+function switch_settings(gradeable_id) {
+    cy.visit(['sample', 'gradeable', gradeable_id, 'update']);
+    cy.get('#page_3_nav').click();
+    cy.get('input[data-testname="grader_assignment_method"][value="1"]').check();
+}
+
+function revert_settings(gradeable_id, setting) {
+    cy.visit(['sample', 'gradeable', gradeable_id, 'update']);
+    cy.get('#page_3_nav').click();
+    cy.get(`input[data-testname="grader_assignment_method"][id=${setting}`).check();
+}
+
+function select_gradeable() {
+    // This gets a gradeable that has been graded already, so there are submissions available.
+    cy.get('[data-testid="grade-table"]').contains('/ 12').click({ force: true });
+    cy.get('[data-testid="show-autograding"]').click();
+    cy.get('[data-testid="show-submission"]').click();
+    cy.get('[data-testid="folders"]').contains('submissions').click();
+    cy.get('#div_viewer_sd1').contains('words_').click();
+    cy.get('#pageContainer1').should('be.visible');
+}
+
+function check_pdf_access(gradeable_id) {
+    cy.visit(['sample', 'gradeable', gradeable_id, 'grading', 'details']);
+    cy.get('#agree-button').click({ force: true });
+    cy.get('[data-testid="details-table"]').should('be.visible');
+    cy.get('[data-testid="view-sections"]').then(($button) => {
+        if ($button.text().includes('View All')) {
+            $button.click();
+        }
+    });
+    select_gradeable();
+}
+
+function minimum_pdf_access(gradeable_id) {
+    cy.visit(['sample', 'gradeable', gradeable_id, 'grading', 'details']);
+    cy.get('#agree-button').click({ force: true });
+    cy.get('[data-testid="details-table"]').should('be.visible');
+    select_gradeable();
+}
```

Comment:
It would be good to add some sort of check to make sure we *don't* see stuff we're not supposed to see.  Otherwise, what makes this different from `check_pdf_access`?  I believe that this test case would pass any time `check_pdf_access` passes.

Reply:
The 'grader' and 'student' users don't have the ability to view different sections, like the TA and Instructor do, and since the TA is not assigned to all of the gradeables, you sometimes have to click the 'view all' button in order to view the table. This function _may_ succeed when check_pdf_access succeeds, however it is for the student and grader users, because the 'view all' or 'view sections' button isn't visible to them.

Reply:
I have added a check to make sure the grader/student _don't_ see the 'view all' button.